lib/util/index.js

```diff
@@ -289,3 +300,4 @@ exports.interceptEvent = interceptEvent;
 exports.legacyPromise = legacyPromise;
 exports.makeUUID = makeUUID;
 exports.proxyProperties = proxyProperties;
+exports.support = support;
```

Comment:
support isn't very discriptive of what this functio does and returns ? I suggest rename it to `isWebRTCSupported`?

Reply:
I guess you are considering  module name (twilio-webrtc) when naming this function, but still it would be more readable to be explicit in the name.

Reply:
@makarandp0 Please see this [comment](https://github.com/twilio/twilio-video.js/pull/1039#discussion_r432727351).